azure-pipelines.yml

```diff
@@ -0,0 +1,96 @@
+
+variables:
+  CurrentSemanticVersionBase: '1.5.3'
+  PreviewNumber: $[counter(variables['CurrentSemanticVersionBase'], 1001)]
+  CurrentSemanticVersion: '$(CurrentSemanticVersionBase)-preview$(PreviewNumber)'
+  NugetPackageVersion: '$(CurrentSemanticVersion)'
+  MONO_VERSION: 6_4_0
+  XCODE_VERSION: 11.1
```

Comment:
11.5 is the current stable version, but I don't know if it's available on Azure

Reply:
Yep! Don't think so, but I'm still working on this one :)

Reply:
Updated to macOS 10.15 and Xcode 11.4 which seems to be the default on the Hosted Mac agents for now